sendgrid-php.php

```diff
@@ -11,10 +11,33 @@
 
 // Can't locate SendGrid\Mail\Mail class?
 if (!class_exists(Mail::class)) {
-    // Suggest to load Composer autoloader of project
+    // if the user is not using composer, register an autoload function to load based on relative directory structure
     if (!$composerAutoloadFileExists) {
-        //  Can't load the Composer autoloader in this project folder
-        error_log("Composer autoloader not found. Execute 'composer install' in the project folder first.");
+        require_once( __DIR__ . '/lib/BaseSendGridClientInterface.php');
+        require_once( __DIR__ . '/lib/SendGrid.php');
+        require_once( __DIR__ . '/lib/TwilioEmail.php');
+
+        // autoload based on directory structure & without composer
+        spl_autoload_register(function($class){
+            if ( starts_with($class, 'SendGrid\\') ){
+                $path = str_replace('\\', '/', $class);
+
+                $pieces = explode('/', $path);
+                array_shift($pieces); // we don't use the first piece (SendGrid)
+                $file = array_pop($pieces);
+
+                $dirPath = __DIR__ . '/lib/' . strtolower(implode('/', $pieces));
+                $filePath = $dirPath . '/' . $file . '.php';
+
+                if ( file_exists($filePath) ) {
```

Comment:
nit: remove spaces around conditional statement in parenthesis for style consistency